test/e2e/bundle_e2e_test.go

```diff
@@ -34,19 +41,49 @@ var _ = Describe("Installing bundles with new object types", func() {
 	})
 
 	When("a bundle with a pdb, priorityclass, and VPA object is installed", func() {
-		By("including the VPA CRD in the CSV")
 		const (
 			packageName = "busybox"
 			channelName = "alpha"
 			subName     = "test-subscription"
 		)
+		var u unstructured.Unstructured
```

Comment:
`u` could be anything, I think it will be easier to read if this variable name is more descriptive